### ISSUE TITLE
Deep biomarkers of human aging: Application of deep neural networks to biomarker development

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#37]
http://www.impactaging.com/papers/v8/n5/full/100968.html

**Comment by sw1** [comment]
## Summary

This a pretty straightforward ML paper. With the aim of predicting chronological age from blood chemistry, they built a bunch of DNNs, optimized them, and reported the results, as well as combining the best performing ones into an ensemble. They also identified important features.

This can clearly be grouped with other EHR papers, probably as a cited example for deep learning's use in a hospital context. I can't imagine anything beyond that, though. 

## Problem

Research has shown that biomarkers of age-associated pathology may reflect senescence modifications and hence may act as aging clocks, but most of these biomarkers are not important in inferring health status and are therefore not frequently collected. 

Here, they tried to use readily collected data (blood chemistry) to predict patient chronological age.

## Methods

### Data 

62,414 patient records containing age, sex, and 41 common blood markers such as glucose, cholesterol, etc. They omitted WBC data because of how variable it is across the general population. All biomarkers were normalized to 0-1.

### Design

They treated this as a regression problem -- not a classification problem -- primarily so they could associate patient age with biomarkers that are correlated. 

4 metrics were evaluated using 90/10 training/testing split: Pearson's correlation, R-squared, MAE, and prediction accuracy coded 0 or 1 where 1 is given if the predicted value falls withing a predefined neighborhood of the true value (+/- 10 years).

They ranked important features in a fashion similar to RFs: they randomly shuffled features and calculated the drop in performance, in terms of R-squared.

40 DNNs were fit with various hyperparameters, and they selected 21 DNNs that performed the best to be combined into an ensemble via stacked generalization (elastic net turned out to be the stacking best model). They selected these 21 DNNs by iteratively adding each DNN prediction vector into the ensemble in an order based on either (1) decreasing R-squared or (2) decreasing correlation. Both methods identified a 21 net ensemble.

They used feedfoward NNs with at least 4 layers. The best performing DNN in their ensemble had 5 hidden layers with 2000, 1500, 1000, 500, and 1 node each, used PReLU, AdaGrad, L2 regularization, and dropout (0.2)

## Results

That best performing DNN had an R-squared of 0.8 and 82% prediction accuracy and outperformed KNNs, SVMs, RFs, among others.

The ensemble had a 0.82 R-squared and a 83.5% prediction accuracy.

They compared their performance to studies that predicted age with transcriptomic biomarkers (R-squared = 0.6) and epigenetic biomarkers (R-squared = 0.93 and R-squared = 0.89).

The feature importance procedure ranked albumin, glucose, ALP, urea and RBCs at the top.

## Notes

The DNN is available here: www.aging.ai

**Comment by agitter** [comment]
DOI link: http://doi.org/10.18632/aging.100968
